wlauto/utils/android.py

```diff
@@ -100,6 +100,15 @@
     'UNINSTALL_SHORTCUT',
 ]
 
+GOOGLE_KEYBOARD_PACKAGE = 'com.android.inputmethod.latin'
+
+# List of known Google Keyboard versions that have problems
+# when key presses are used in AndroidUiAutoBenchmark workloads
+GOOGLE_KEYBOARD_BLACKLIST = [
+    '5.0.25.122319759-armeabi-v7a',
```

Comment:
we should probably only go down to major.minor.path with the version numbers, things like the ABI at the end will change but it will still have the same bug